packages/palette/src/elements/Input/Input.tsx

```diff
@@ -123,11 +161,24 @@ const StyledInput = styled.input<StyledInputProps>`
       ${props.active && INPUT_STATES.active}
       ${props.disabled && INPUT_STATES.disabled}
       ${props.error && INPUT_STATES.error}
+      ${!!props.prefixOffset &&
+      css`
+        padding-left: ${props.prefixOffset}px;
+      `}
+      ${!!props.suffixOffset &&
+      css`
+        padding-right: ${props.suffixOffset}px;
+      `}
```

Comment:
I'm going to need some kind of visual explanation for the existence of this?

Reply:
This will be removed